arangod/Network/Utils.cpp

```diff
@@ -42,21 +42,6 @@
 
 namespace arangodb::network {
 
-Headers addAuthorizationHeader(
-    std::unordered_map<std::string, std::string> const& originalHeaders) {
-  auto auth = AuthenticationFeature::instance();
```

Comment:
nit: with the function being removed, we can now also remove the includes at the top of the file for the AuthenticationFeature and the TokenCache.